ha/test/integration/health_monitor/test_publisher.py

```diff
@@ -45,9 +45,12 @@ def receive(message):
         state = "failed"
         message_type = event_manager.subscribe('csm', [SubscribeEvent(resource_type, [state])])
         print(f"Subscribed {component}, message type is {message_type}")
```

Comment:
I think its meant for RecoveryActionEvent that we send it to HARE. So, the structure is different. We don't need header and payload. Right @ArchanaLimaye 
Please change as discussed.